telegram/ext/filters.py

```diff
@@ -133,7 +133,14 @@ def __init__(self, name: str = None, data_filter: bool = False):
 
     @abstractmethod
     def check_update(self, update: Update) -> Optional[Union[bool, DataDict]]:
-        ...
+        if (  # Only message updates should be handled.
+            update.channel_post
+            or update.message
+            or update.edited_channel_post
+            or update.edited_message
+        ):
+            return True
+        return False
```

Comment:
A unit test for this would be good :)

Reply:
This is implicitly tested in [`test_messagehandler`](https://github.com/python-telegram-bot/python-telegram-bot/blob/54ed706df44eff2d2f3db5fbba15e2be475e4df6/tests/test_messagehandler.py#L131) already, i'll add it in anyway